tools/genomespace/genomespace_importer.xml

```diff
@@ -1,26 +1,37 @@
 <?xml version="1.0"?>
-<tool name="GenomeSpace Importer" id="genomespace_importer" tool_type="data_source" force_history_refresh="True" hidden="True" display_interface="False" require_login="True" version="0.0.2">
+<tool name="GenomeSpace Importer" id="genomespace_importer" require_login="False" version="0.0.4">
     <description> - receive data from GenomeSpace</description>
-    <command interpreter="python">genomespace_importer.py
-        --genomespace_site "prod"
-        #assert $__user__, Exception( 'You must be logged in to use this tool.' )
-        #set $username = $__user__.preferences.get( 'genomespace_username', None )
-        #set $token = $__user__.preferences.get( 'genomespace_token', None )
-        #assert None not in ( $username, $token ), Exception( 'You must associate a GenomeSpace OpenID with your account and log in with it.' )
-        --username "${username}"
-        --token "${token}"
+    <command><![CDATA[
+        export PYTHONPATH=$__root_dir__/lib:\$PYTHONPATH &&
```

Comment:
you probably want to replace this and the below line to python `$__tool_directory__/genomespace_importer.py`